sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/ClassNameTest.cs

```diff
@@ -44,8 +44,12 @@ public void ClassName_CSharp()
                     @"TestCases\ClassName.cs",
                     @"TestCases\ClassName.Partial.cs",
                 }, new CSharp.ClassAndMethodName(),
+#if NETFRAMEWORK
                 options: ParseOptionsHelper.FromCSharp8,
                 additionalReferences: NuGetMetadataReference.NETStandardV2_1_0);
+#else
+                options: ParseOptionsHelper.FromCSharp8);
+#endif
```

Comment:
We can simplify this by adding `additionalReferences: NuGetMetadataReference.NETStandardV2_1_0` only for `NETFRAMEWORK`?

Since for named parameters the order does not matter we can have something like:
```
#if NETFRAMEWORK
	additionalReferences: NuGetMetadataReference.NETStandardV2_1_0,
#endif
	options: ParseOptionsHelper.FromCSharp8);
```

Another option would be to add a new method in `MetadataReferenceFacade` to get a reference to .Net Standard.

Edit: there are multiple occurrences of this. We might want to check where this applies and simplify them.